docs/spark-connector/spark-connector.md

```diff
@@ -18,13 +18,17 @@ The Gravitino Spark connector leverages the Spark DataSourceV2 interface to faci
 
 ## Requirement
 
-* Spark 3.4
-* Scala 2.12
-* JDK 8,11,17
+* Spark 3.3 or 3.4 or 3.5
+* Scala 2.12 or 2.13
+* JDK 8 or 11 or 17
+
+:::info
+Gravitino spark connector doesn't support scala 2.13 for Spark3.3.
```

Comment:
it seems to be better placed in [how-to-build](https://github.com/datastrato/gravitino/pull/3614/files#diff-ff84168c4ba974b82b38aa13f7e311aaf05a3212a3e56b8af39e83f94b95e90c) doc, WDYT?
to remind users not to use scala2.13 when compiling spark3.3

Reply:
done